src/pages/Landing/components/NavBar/NavBar.module.scss

```diff
@@ -1,49 +1,81 @@
 // Loading variables for styling
 
 @import '../../../../assets/styles/sass/variables';
+@import '../../../../assets/styles/sass/mixin';
+
 
 .Navigation {
+  position: relative;
   box-sizing: content-box;
   top: 0;
   left: 0;
+  width: 100%;
   font-family: $font-secondary;
-  height: 4.4rem;
-  display: flex;
-  padding: 3.4rem 4.4rem 0;
-  align-content: center;
-  color: #ffffff;
+  background-color: $primary-color;
+  color: #ffffff !important;
```

Comment:
1. use a variable for color. 
2. why there is a need to use !important can this be solved with [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)